src/cc65/typeconv.c

```diff
@@ -167,6 +167,11 @@ static void DoConversion (ExprDesc* Expr, const Type* NewType)
 ExitPoint:
     /* The expression has always the new type */
     ReplaceType (Expr, NewType);
+
+    /* Bit-fields are converted to integers */
+    if (ED_IsBitField (Expr)) {
+        ED_DisBitField (Expr);
```

Comment:
The test isn't needed; `ED_DisBitField()` changes nothing when the expression isn't a bit-field.

Reply:
Removed.